src/sentry/runner/commands/presenters/presenterdelegator.py

```diff
@@ -7,7 +8,7 @@ def __init__(self) -> None:
         self._consolepresenter = ConsolePresenter()
 
         self._slackpresenter = None
-        if SlackPresenter.is_slack_enabled():
+        if options.get("options_automator_slack_webhook") and SlackPresenter.is_slack_enabled():
```

Comment:
Move the check inside `is_slack_enabled` ? 
THis is part of the condition that tells you if slack is enabled.